pkg/types/route.go

```diff
@@ -75,7 +75,7 @@ const (
 	HandlerStop
 )
 
-const DefaultRouteHandler = "default"
```

Comment:
maybe it should not be exposed by variable, but by methods

Reply:
I'm not sure introduce method in types package is a good idea.
or we could introduce a new global variable somewhere?

Reply:
we could provide a new variable which has high priority, if it unset, using `DefaultRouteHandler` as default